pkg/madmin/heal-commands.go

```diff
@@ -196,8 +203,12 @@ func (adm *AdminClient) Heal(bucket, prefix string, healOpts HealOpts,
 		queryVals.Set("clientToken", clientToken)
 		body = []byte{}
 	}
+
+	// Anyone can be set, either force start or forceStop.
 	if forceStart {
 		queryVals.Set("forceStart", "true")
+	} else if forceStop {
+		queryVals.Set("forceStop", "true")
```

Comment:
I think we can help programming errors to be caught sooner by refusing to send the request when both are set.

Reply:
@donatello that can be done.

Reply:
done